.github/workflows/test.yml

```diff
@@ -10,10 +10,11 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
The `ubuntu-latest` image now points to ubuntu-24.04 which does not support Python 3.7 and the build fails while setting up Python 

https://github.com/django-notifications/django-notifications/actions/runs/14221321649/job/39849812059#step:4:8